db/seeds.rb

```diff
@@ -1,7 +1,31 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+# Create 10 Random Users. User will be created with default role
+
+puts "Seeding Users"
+
+10.times do
+  User.create!(
+    name: Faker::Name.name,
+    email: Faker::Internet.email,
+    password: "123456",
+    confirmed_at: Date.today
+  )
+end
+
+# Collect all the randomly seeded users
+users = User.all
+
+puts "Seeding Wikis"
+
+# Create 20 Random Wikis.
+20.times do
+  Wiki.create!(
+    title: Faker::Lorem.sentence(6),
+    body: Faker::Lorem.paragraph(4),
+    user_id: users.sample.id,
+    private: false
```

Comment:
It is, but I might eventually change that to a random as well once I get premium/privates set up in one of the following user stories, so I kept it there.